languages/elixir/exercises/concept/charlists/.docs/introduction.md

```diff
@@ -0,0 +1,57 @@
+Charlists are created using single quotes.
+
+```elixir
+'hello'
+```
+
+A charlist is a list of integers where the integers represent the Unicode values of a given character — also known as code points.
```

Comment:
Somehow I expected the second line ("A charlist is...") to be the first line, as it tells me what charlists are. But feel free to ignore :)

Reply:
I totally agree that with that phrasing, the lines seem out of order. But I believe the example of a charlist using single quotes needs to go before any mention of lists of integers, so I rephrased the intro a bit to make more sense.